openwisp_monitoring/device/tests/test_check_signals.py

```diff
@@ -0,0 +1,48 @@
+from django.test import TestCase
+from swapper import load_model
+
+from openwisp_monitoring.device.models import CHECK_CLASSES
+
+Check = load_model('check', 'Check')
+Device = load_model('config', 'Device')
+DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
+
+
+class TestCheckSignals(TestCase):
+    def setUp(self):
+        self.device = self._create_device(organization=self._create_org())
+        self.ping_check = Check.objects.create(
+            name='Ping Check',
+            check_type=CHECK_CLASSES[0][0],
+            content_object=self.device,
+            params={},
+        )
+
+    def test_check_signals(self):
+        with self.subTest('Test disabling a critical check'):
+            self.ping_check.is_active = False
+            self.ping_check.save()
```

Comment:
Let's assert that the `health_status_change` signal has been sent as expected.

Look for `catch_signal` in the codebase of this repo to see how to do that, it's also documented in the openwisp-utils README (please read it).